config/docker-dev.yml

```diff
@@ -5,6 +5,11 @@ analysis:
       rpc: unix:/node/data/internal.sock
       chaincontext: b11b369e0da5bb230b220127f5e7b242d385ef8c6f54906243f30af63c815535
       from: 8048956
+    - name: emerald_main_damask
+      chain_id: oasis-3
```

Comment:
Chain ID is used for Ethereum chains to prevent [cross-chain replay attacks](https://eips.ethereum.org/EIPS/eip-155). It's a different chain ID from the chain ID as specified by our consensus layer -- which is used to denote versions between dump&restore (breaking) upgrades at the consensus layer.

Dump&restore upgrades are _technically_ entirely independent blockchains and can have entirely different parameters and semantics, but with version `n` being given genesis state that was dumped after the last block of version `n-1`.